src/renderer/replugged.ts

```diff
@@ -23,6 +23,11 @@ export * as components from "./modules/components";
 export { default as notices } from "./apis/notices";
 export { default as commands } from "./apis/commands";
 export * as settings from "./apis/settings";
+export namespace api {
+  export let messagePopover: EventTarget;
+}
+import { MessagePopoverAPI } from "./apis/message-popover";
+api.messagePopover = new MessagePopoverAPI();
```

Comment:
Import this above the namespace and export it directly in the namespace instead of assigning it after the fact

Reply:
Lol I had this already, I just forgot to commit it